.github/workflows/ci-build.yml

```diff
@@ -72,9 +72,19 @@ jobs:
           name: build-log-java-${{ matrix.java }}-${{ matrix.os }}
           path: build.log
 
+      - name: Change Files
+        run: |
+          echo modification >> CONTRIBUTING.md
+          touch new-file
+          mkdir new-dir
+          touch new-dir/new-file
+          rm README.md
+
       - name: Verify Changed Files
-        uses: tj-actions/verify-changed-files@v20
         id: verify-changed-files
+        run: |
+          changed_files=$(git diff --name-only ${{ github.sha }}; git ls-files --others --exclude-standard | tr '\n' ' ')
```

Comment:
```suggestion
          changed_files=$((git diff --name-only ${{ github.sha }}; git ls-files --others --exclude-standard) | tr '\n' ' ')
```

Reply:
Using `&&` instead of the `;` would in case the first command fails.

Reply:
And you need to set `set -o pipefail` otherwise errors are also hidden due to the pipe.

Reply:
Doesn't seem to fix the issue. :face_with_diagonal_mouth: